requirements.txt

```diff
@@ -3,8 +3,10 @@ django-debug-toolbar==3.4.0
 djangorestframework==3.13.1
 djangorestframework-simplejwt==5.2.0
 drf-spectacular==0.22.1
-Pillow==9.1.1
+Pillow
 flake8==5.0.4
 flake8-quotes==3.3.1
 flake8-variables-names==0.0.5
-pep8-naming==0.13.2
\ No newline at end of file
+pep8-naming==0.13.2
+python_dotenv==1.0.0
+psycopg2
```

Comment:
In `requirements.txt` it is a better practice to include a specific version to ensure package compatibility. Add versions to the `requirements.txt`.

Reply:
done